workspaces/cli-shared/src/captures/avro/file-system/capture-saver-with-diffs.ts

```diff
@@ -103,25 +102,6 @@ export class CaptureSaverWithDiffs extends FileSystemAvroCaptureSaver {
       );
     }
 
-    if (this._config.shouldCollectCoverage) {
```

Comment:
You should clean up init() as well; that should allow you to get rid of the useoptic/domain dependency from this file

Reply:
Love it. Done.